onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2363,6 +2363,9 @@ static constexpr OrtApi ort_api_1_to_9 = {
     &OrtApis::GetSparseTensorValues,
     &OrtApis::GetSparseTensorIndicesTypeShape,
     &OrtApis::GetSparseTensorIndices,
+    // End of Version 9 - DO NOT MODIFY ABOVE (see above text for more information)
```

Comment:
Line 2343 "// Version 9 - In development, feel free to add/remove/rearrange here" should be removed since ver 9 will be sealed.

Reply:
fixed